apix-docker/tomcat/context.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Context allowCasualMultipartParsing="true" allowMultipleLeadingForwardSlashInPath="true">
```

Comment:
please add some information in comment why are we overriding this, including the issue with the multipleleadingforwardslash